__tests__/cache.test.js

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable jest-formatting/padding-around-test-blocks */
+const nock = require('nock');
+const fsMock = require('mock-fs');
+const findCacheDir = require('find-cache-dir');
+const { join } = require('path');
+const fs = require('fs').promises;
+
+const Cache = require('../src/cache');
+const pkg = require('../package.json');
+
+let readmeExampleJson;
+let readmeExampleYaml;
+const examplesDir = join(__dirname, 'examples');
+
+expect.extend({
+  // Custom matcher so we can easily test that dereferencing of OpenAPI files is working as expected.
+  toBeDereferenced(received) {
+    const pass = !received.filter(obj => '$ref' in obj).length;
+    if (pass) {
+      return {
+        message: () => `expected supplied array not to be dereferenced`,
+        pass: true,
+      };
+    }
+
+    return {
+      message: () => `expected supplied array to be dereferenced`,
+      pass: false,
+    };
+  },
+});
+
+beforeEach(async () => {
+  readmeExampleJson = await fs.readFile(
+    join(__dirname, '../node_modules/@readme/oas-examples/3.0/json/readme.json'),
```

Comment:
You should be able to do this using a `require.resolve('@readme/oas-examples/3.0/json/readme.json')`

Reply:
TIL about `require.resolve`